plugins/woocommerce/src/Internal/Admin/Marketplace.php

```diff
@@ -91,6 +91,17 @@ private static function get_marketplace_update_count_html() {
 			++$count;
 		}
 
+		return $count;
+	}
+
+	/**
+	 * Create the menu bubble for extensions menu based on number of updates available.
+	 *
+	 * @return string
+	 */
+	private static function get_marketplace_update_count_html() {
```

Comment:
Why have we copied this method out of the Updater instead of updating the Updater class to support WUM? From what I can tell both this method and `get_marketplace_update_count` could be supported by minor updates to the underlying class?

Reply:
Thanks for the feedback @laurendavissmith. This totally makes sense. I've done some refactoring here (https://github.com/woocommerce/woocommerce/pull/46088/commits/95f53bc75f42767fadbc987879136bfd1e3db869) to remove unwanted methods from `\Automattic\WooCommerce\Internal\Admin\Marketplace`